doc/changes/changes_1.2.0.md

```diff
@@ -1,15 +1,48 @@
-# Cloud Storage Extension 1.2.0, released 2021-MM-DD
+# Cloud Storage Extension 1.2.0, released 2021-06-15
 
-Code name:
+Code name: Fixed bug listing files on a path using asterisk
 
 ## Summary
 
-## Bugs
+In this release we fixed a bug related to listing files using asterisk on a
+path. We also updated user guide and added link checker to detect broken links
+in the project documentation.
```

Comment:
It will be copied as it is to the community (3 lines). Is it okay?

Reply:
Ahh yes, I forget it :)

Reply:
Fixed.